### PR TITLE
#1109: Improve Pit Coverage

### PR DESCRIPTION
**Pull request by goldbal330**
Signed-off-by: Alexander Goldberg <alexander.goldberg@bnymellon.com>

### CODE REVIEW

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java [hunk 1]
```diff
@@ -1402,4 +1460,62 @@ public void classIsNonInstantiable()
...
+                ArrayIterate.min(integerTwins, Comparators.byFunction(Functions.firstOfPair())));
+
+        Assert.assertEquals(Tuples.twin(9, 1),
+                ArrayIterate.min(integerTwins, Comparators.byFunction(Functions.secondOfPair())));
+    }
+
+    @Test
+    public void min()
+    {
+        Assert.assertEquals((Integer) 1, ArrayIterate.min(3, 1, 20));
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java:1490]: Replace with `Integer.valueOf(1)`.

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java [hunk 2]
```diff
@@ -1402,4 +1460,62 @@ public void classIsNonInstantiable()
...
+    }
+
+    @Test
+    public void max()
+    {
+        Assert.assertEquals((Integer) 20, ArrayIterate.max(3, 1, 20));
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java:1496]: Replace with `Integer.valueOf(20)`.

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ListIterateTest.java [hunk 3]
```diff
@@ -411,6 +621,25 @@ private void assertReverseForEachWithIndex(List<Integer> list)
         });
     }
 
+    @Test
+    public void forEach()
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/utility/ListIterateTest.java:625]: Rename the test `forEachInRange`, even though the method is an overloaded of `forEach`.

**Review by donraab — CHANGES_REQUESTED** [review]
Thanks for the contribution @goldbal330. I've requested a few changes.

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java (hunk 2 above)
Comment (goldbal330) [reply unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java]: Fixed.

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java (hunk 1 above)
Comment (goldbal330) [reply unit-tests/src/test/java/org/eclipse/collections/impl/utility/ArrayIterateTest.java]: Fixed.

File: unit-tests/src/test/java/org/eclipse/collections/impl/utility/ListIterateTest.java (hunk 3 above)
Comment (goldbal330) [reply unit-tests/src/test/java/org/eclipse/collections/impl/utility/ListIterateTest.java]: Done.

**Review by donraab — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**